common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1613,7 +1613,7 @@ def get_course_template_context(self):
         request.user = user
         user_info = cc.User.from_django_user(self.runtime.user).to_dict()
         course_id = self.runtime.course_id
-        course = get_course_with_access(self.runtime.user, 'load_forum', course_id)
+        course = get_course_with_access(self.runtime.user, 'load', course_id, check_if_enrolled=True)
         user_cohort_id = get_cohort_id(user, course_id)
 
         unsafethreads, query_params = get_threads(request, course_id)
```

Comment:
This is passing in a `course_id` but the `get_threads` code seems to expect a `course` object (though it only really needs an ID).

As a result, I'm seeing:
```
  File "/edx/app/edxapp/edx-platform/common/lib/xblock/discussion/discussion_forum/discussion_forum.py", line 150, in student_view
    fragment = self._student_view_lms()
  File "/edx/app/edxapp/edx-platform/common/lib/xblock/discussion/discussion_forum/discussion_forum.py", line 160, in _student_view_lms
    context = discussion_service.get_course_template_context()
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/x_module.py", line 1619, in get_course_template_context
    unsafethreads, query_params = get_threads(request, course_id)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/newrelic-2.46.0.37/newrelic/api/function_trace.py", line 104, in literal_wrapper
    return wrapped(*args, **kwargs)
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/django_comment_client/forum/views.py", line 113, in get_threads
    'course_id': unicode(course.id),
AttributeError: 'CourseLocator' object has no attribute 'id'
```